docker-compose.yml

```diff
@@ -3,9 +3,13 @@ version: '3.8'
 services:
 
   backend:
-    build: ./backend
+    build:
+      context: ./backend
+      args:
+        CONFIG_FILE: config.yml # Replace with your own config file
```

Comment:
This would require rebuilding if you change your config, no? Would it be better to instead have a volume point to the config on disk?

Reply:
Ok, I'll make the necessary changes 👍